README.md

```diff
@@ -0,0 +1,97 @@
+# RyanAirTestAutomation
+**PythonTestAutomationFramework**
+
+**Description**
+
+Test Automation Framework for web automation of  "https://www.ryanair.com".
+
+Main technologies are used in project:
+
+- Python == 3.10
+- pytest == 6.2.5
+- Selenium == 3.141.0
+
+* Framework is based on Page Object Model. 
+* Reporting implemented by using Allure report.
+* Logging to external files implemented using Loguru
+
+The project is being developed during the iTechArt internship.
+
+# How to install it
+
+Make sure you have python3.10 installed on your machine by typing in cmd 
+
+    ``$ python3 --version`` 
+
+if not - go to https://realpython.com/installing-python/#step-1-download-the-python-3-installer.
+
+1)  Clone repository: 
+
+    ``$ clone https://github.com/TA-user/RyanAir_Test_Automation.git``
+
+2)  You have to install allure command line and add the allure folder installation into system environment variable: https://docs.qameta.io/allure/#_installing_a_commandline
+
+3)  Make sure you have pipenv by typing in cmd 
+  
+    ``$ pipenv --version`` 
+  
+If not - you have to install pipenv for creation virtual environment and installation packages: https://pipenv.pypa.io/en/latest/
+
+    ``$ pip install --user pipenv``
+
+4)  Install dependencies:
+
+    ``$ pipenv install``
+
+5)  Add your own credentials (**USERNAME** and **PASSWORD**) for logging to https://www.ryanair.com in ``config.py``:
```

Comment:
Maybe we should say that you can enter username and password from command line?

Reply:
Agree

Reply:
fixed